dash/testing/browser.py

```diff
@@ -88,15 +78,8 @@ def __enter__(self):
     def __exit__(self, exc_type, exc_val, traceback):
         try:
             self.driver.quit()
-            if self._percy_run and self._percy_finalize:
```

Comment:
shall we remove the `percy_run` and `percy_finalize` args, and the `--nopercyfinalize` pytest option?